src/main/c/src/SerialImp.c

```diff
@@ -5117,7 +5116,7 @@ JNIEXPORT void JNICALL RXTXPort(interruptEventLoop)(JNIEnv *env,
 	}
 	index->closing = 1;
 #endif
-	report("interruptEventLoop: interrupted\n");
+	report_verbose("InterruptEventLoop interrupted");
```

Comment:
I think changing the case of the method name here might've been a typo?

Reply:
Well, I tried to make all starting characters upper case. IDK if that is reasonable here.

Reply:
I like it in the other cases, but I think this one should stay lower-case because it's a reference to its own method name. Alternatively, you could use a real English sentence: e.g., “Event loop interrupted”.

Reply:
I think a real sentence is a good idea :D